dll/win32/browseui/CAutoComplete.cpp

```diff
@@ -302,12 +302,15 @@ static void Edit_BackWord(HWND hwndEdit)
         return;
     }
 
-    for (; 0 < iStart; --iStart)
+    WORD wType1, wType2;
+    for (--iStart; 0 < iStart; --iStart)
     {
-        WCHAR ch1 = pszText[iStart - 1];
-        WCHAR ch2 = pszText[iStart];
-        if ((wcschr(L"\\/.:;", ch1) && ch2 && !IsCharSpaceW(ch2)) ||
-            (IsCharSpaceW(ch1) && IsCharAlphaNumericW(ch2)))
+        WCHAR& ch1 = pszText[iStart - 1];
+        WCHAR& ch2 = pszText[iStart];
+        GetStringTypeW(CT_CTYPE1, &ch1, 1, &wType1);
+        GetStringTypeW(CT_CTYPE1, &ch2, 1, &wType2);
```

Comment:
Why not directly pass `pszText + iStart` into `GetStringTypeW` ?
It seems pretty weird to make `ch1` and `ch2` a reference to an entry in an array?

Reply:
Fixed. OK.